conf/airframes/TUDELFT/tudelft_ardrone2_opticflow_stereo.xml

```diff
@@ -23,8 +23,8 @@
 
   <modules main_freq="512">
     <module name="bat_voltage_ardrone2"/>
-    <module name="cv_opticflow"/>
-    <module name="opticflow_hover"/>
+    <!--module name="cv_opticflow"/>
+    <module name="opticflow_hover"/-->
```

Comment:
true, but it was just that I noticed this airframe file needed some updating, so that's why this quick unrelated change.